regolith/helpers/a_todohelper.py

```diff
@@ -0,0 +1,126 @@
+"""Helper for adding a projectum to the projecta collection.
+
+   Projecta are small bite-sized project quanta that typically will result in
+   one manuscript.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "todolist"
+Importance = [0, 1, 2, 3, 4, 5]
+
+
+def subparser(subpi):
+    subpi.add_argument("description", help="the description of the to_do task",
+                       default=None)
+    subpi.add_argument("-i", "--id", help="ID of the to_do list's owner. The default id is saved in user.json. ")
+    subpi.add_argument("-d", "--due_date",
+                       help="Proposed due date for the to_do task."
+                       )
+    subpi.add_argument("-p", "--importance",
+                       help=f"The importance of this task from {Importance}. Default is 0 (unimportant).",
+                       default=0
```

Comment:
default importance should probably be somewhere in the middle maybe but perhaps we need to start using this to learn how to do this best.  I am guessing something less granular might be better 0,1,2 but we will see.